models/public/single-human-pose-estimation-0001/description/single-human-pose-estimation-0001.md

```diff
@@ -0,0 +1,57 @@
+# single-human-pose-estimation-0001
+
+## Use Case and High-Level Description
+Single human pose estimation model based on https://arxiv.org/pdf/1906.04104.pdf.
+
+## Specification
+
+| Metric                                                        | Value                   |
+|---------------------------------------------------------------|-------------------------|
+| AP(coco orig)                                                 | 68                      |
+| GFlops                                                        | 60.125                  |
+| MParams                                                       | 33.165                  |
+| Source framework                                              | PyTorch\*               |
+
+
+## Inputs
+
+### Original model
+
+Name: "data" , shape: [1x3x384x288] - An input image in the format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - RGB. Mean values - [123.675,116.28,103.53]. Scale values - [58.395,57.12,57.375]
+
+### Converted model
+
+Name: "data" , shape: [1x3x384x288] - An input image in the format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - BGR.
+
+## Outputs
+
+### Original model
+
+The net outputs list of tensor. Count of list elements is 6. Every tensor with shapes: [1x17x48x36]. ( For every keypoint own heatmap)
+
+
+### Converted model
+
+The net outputs tensor with shapes: [1x17x48x36]. ( For every keypoint own heatmap)
+
+## Legal Information
+[LICENSE](https://raw.githubusercontent.com/opencv/openvino_training_extensions/develop/LICENSE)
+
+[*] Other names and brands may be claimed as the property of others.
+[**] The six outputs are necessary in order to calculate the loss in during training. But in the future, for obtaining the results of prediction and postprocessing them, the last output is used.
```

Comment:
This footnote isn't referenced anywhere. Also, it's not legal information.

Reply:
Ok, where should i write this information?

Reply:
Just move it into the output section.